posts/2020/08/reproducible-builds.md

```diff
@@ -0,0 +1,146 @@
+<!--
+.. title: IPython reproducible build
+.. slug: ipython-reproducible-builds
+.. date: 2020-08-05 22:00:00 UTC-00:00
+.. author: Matthias Bussonnier
+.. tags: Labs, IPython, reproducible-builds, packaging
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+Starting with IPython 7.16.1 (released in June 2020), you _should_ be able to recreate the sdist (`.tar.gz`) and wheel
+(`.whl`), and get bytes for bytes identical result to the ones publish on PyPI. This is a critical step toward being able
+to _trust_ your computing platforms, and a key component to improve efficiency of build and packaging platform, with
+potentially impacts on fast conda environment creation for users. The following goes into some reason of why you should care.
+
+<!-- TEASER_END -->
+
+Since the cornerstone Paper [Refections on trusting Trust][1], there are always been advocate of reproducible builds, in
+todays highly interconnected world, and with the speed at which new software is released and deployed, being able to confirm
+the provenance and verify that supply chain has not been affected by malicious actor is often critical. To help in this
+endeavour, the movement of [reproducible builds][2], attempt to push software toward a deterministic and reproducible
+build process.
+
+[2]: https://reproducible-builds.org/
+[1]: https://www.cs.cmu.edu/~rdriley/487/papers/Thompson_1984_ReflectionsonTrustingTrust.pdf
+
+While security is one of the earliest concepts that advocated for reproducible build, there are a number of other
```

Comment:
"While security is one of the earliest concepts that advocated for reproducible build" -> "While information security practitioners were one of the earliest groups who advocated for reproducible builds"